bin/install_deps.sh

```
@@ -0,0 +1,194 @@
+fi
+
+# Initialize rbenv
+if [[ "$OS" == "macos" ]]; then
+  # The rbenv init command adds the config to ~/.zshenv,
+  # which has a source order issue. So we add it manually to
+  # ~/.zshrc instead if not present.
+  # See: https://github.com/rbenv/rbenv?tab=readme-ov-file#how-rbenv-hooks-into-your-shell
+  if ! grep -q "rbenv init" ~/.zshrc; then
+    echo 'eval "$(rbenv init -)"' >> ~/.zshrc
+  fi
+  source ~/.zshrc
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)